tools/release-scripts/tag-and-publish.js

```diff
@@ -1,10 +1,17 @@
 import { safeExec } from '../exec';
+import semver from 'semver';
 import tag from './tag';
 
 export default (version) => {
   console.log('Releasing: '.cyan + 'npm module'.green);
+  let parsed = semver.parse(version);
+  let additionalPublishArgs = '';
+
+  if (parsed.prerelease.length > 0) {
+    additionalPublishArgs = `--tag ${parsed.prerelease[0]}`;
+  }
 
   return tag(version)
-    .then(() => safeExec('npm publish'))
+    .then(() => safeExec(`npm publish ${additionalPublishArgs}`))
     .then(() => console.log('Released: '.cyan + 'npm module'.green));
```

Comment:
You won't like me, but following your remark from yesterday, shouldn't you import `colors` here as well? :mag: